packages/oauth-providers/vitest.config.ts

```diff
@@ -2,6 +2,7 @@ import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
   test: {
+    environment: 'miniflare',
```

Comment:
Using `vitest-environment-miniflare` will make the `crypto` global accessible in tests, similar to what `jest-environment-miniflare` was doing previously.

The tests for `@hono/oauth-providers` use `msw` which won't work with `@cloudflare/vitest-pool-workers`. I did try that first, but it will require rewriting the `msw` handlers to use [`import { fetchMock } from "cloudflare:test"`](https://developers.cloudflare.com/workers/testing/vitest-integration/test-apis/#cloudflaretest-module-definition)